build.gradle.kts

```diff
@@ -164,9 +169,40 @@ tasks.register<Exec>("gitTag") {
     commandLine("git", "tag", releaseTagName)
 }
 
-tasks.register<Exec>("gitPushTag") {
+val gitPushTag = tasks.register<Exec>("gitPushTag") {
     mustRunAfter("publishAllPublicationsToGradleBuildInternalRepository")
     dependsOn("gitTag")
     commandLine("git", "push", "https://bot-teamcity:${project.property("githubToken")}@github.com/gradle/gradle-profiler.git", releaseTagName)
 }
 
+sdkman {
+    api = "https://vendors.sdkman.io"
+    candidate = "gradleprofiler"
+    hashtag = "#gradleprofiler"
+    version = project.version.toString()
+    url = project.gradleInternalRepositoryUrl().resolve("org/gradle/profiler/gradle-profiler/$version/gradle-profiler-$version.zip").toString()
+    consumerKey = project.findProperty("sdkmanKey") as String?
+    consumerToken = project.findProperty("sdkmanToken") as String?
+}
+
+tasks.withType<SdkmanVendorBaseTask>().configureEach {
+    mustRunAfter(gitPushTag)
+}
+
+tasks.withType<SdkDefaultVersionTask>().configureEach {
+    mustRunAfter(tasks.withType<SdkReleaseVersionTask>())
+}
+
+tasks.withType<SdkAnnounceVersionTask>().configureEach {
+    mustRunAfter(tasks.withType<SdkReleaseVersionTask>())
+}
+
+tasks.register("releaseToSdkMan") {
+    val versionString = project.version.toString()
+    val isFinalRelease = Regex("""[0-9\.]*""").matchEntire(versionString) != null
```

Comment:
This could use a bit of commenting. And it would be great to mention the release logic and the fact that we are publishing to SDKMAN in the `RELEASING.md`. That file is in dire need of updating anyway.

Reply:
I added a comment and updated the release instructions.

Reply:
Thanks. I think the way the version number format controls the SDKMAN publishing behavior should also be mentioned in `RELEASING.md`.

Reply:
I added it the releasing instructions as well.